File: sprint2/By languages/python/lesson10code1.py
def func(step):
    print('Call me. Step:', step)

func(1)
func(2)
func(3)

def say_hello(name):
    print(f"Привет, {name}")
    print_horoscope(name.upper())
    print(f"Пока, {name}, хорошего дня!")

def print_horoscope(name):
    print(f"{name}! Сегодня подходящий день для прогулок в парке и изучения рекурсии") 

say_hello('Гоша')

File: sprint2/By languages/C#/lesson10code1.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Func(1);
        Func(2);
        Func(3);

        SayHello("Гоша");
    }

    static void Func(int step)
    {
        Console.WriteLine("Call me. Step: " + step);
    }

    static void SayHello(string name)
    {
        Console.WriteLine("Привет, " + name);
        PrintHoroscope(name.ToUpper());
        Console.WriteLine("Пока, " + name + ", хорошего дня!");
    }

    static void PrintHoroscope(string name)
    {
        Console.WriteLine(name + "! Сегодня подходящий день для прогулок в парке и изучения рекурсии");
    }
}
